test/steps/kogitoapp.go

```diff
@@ -60,82 +71,43 @@ func registerKogitoAppSteps(s *godog.Suite, data *Data) {
 }
 
 // Deploy service steps
-func (data *Data) deployQuarkusExampleServiceWithNative(contextDir, native string) error {
-	kogitoApp := getKogitoAppExamplesStub(data.Namespace, contextDir, framework.MustParseEnabledDisabled(native), false, false)
-	kogitoApp.Spec.Runtime = v1alpha1.QuarkusRuntimeType
-
-	return framework.DeployService(data.Namespace, framework.GetDefaultInstallerType(), kogitoApp)
-}
-
-func (data *Data) deployQuarkusExampleServiceWithNativeAndLabels(contextDir, native string, dt *gherkin.DataTable) error {
-	labels, err := assist.ParseMap(dt)
+func (data *Data) deployQuarkusExampleServiceWithConfiguration(contextDir string, table *gherkin.DataTable) error {
+	kogitoApp, err := getKogitoAppExamplesStub(data.Namespace, contextDir, table)
 	if err != nil {
 		return err
 	}
 
-	kogitoApp := getKogitoAppExamplesStub(data.Namespace, contextDir, framework.MustParseEnabledDisabled(native), false, false)
 	kogitoApp.Spec.Runtime = v1alpha1.QuarkusRuntimeType
-	kogitoApp.Spec.Service.Labels = labels
 
 	return framework.DeployService(data.Namespace, framework.GetDefaultInstallerType(), kogitoApp)
 }
 
-func (data *Data) deployQuarkusExampleServiceWithNativeAndPersistence(contextDir, native string) error {
-	kogitoApp := getKogitoAppExamplesStub(data.Namespace, contextDir, framework.MustParseEnabledDisabled(native), true, false)
-	kogitoApp.Spec.Runtime = v1alpha1.QuarkusRuntimeType
-
-	return framework.DeployService(data.Namespace, framework.GetDefaultInstallerType(), kogitoApp)
-}
-
-func (data *Data) deployQuarkusExampleServiceWithNativeAndPersistenceAndEvents(contextDir, native string) error {
-	kogitoApp := getKogitoAppExamplesStub(data.Namespace, contextDir, framework.MustParseEnabledDisabled(native), true, true)
-	kogitoApp.Spec.Runtime = v1alpha1.QuarkusRuntimeType
-
-	return framework.DeployService(data.Namespace, framework.GetDefaultInstallerType(), kogitoApp)
-}
-
-func (data *Data) deployQuarkusExampleServiceWithBuildResources(contextDir string, dt *gherkin.DataTable) error {
-	resources, err := assist.ParseMap(dt)
+func (data *Data) deploySpringBootExampleServiceWithConfiguration(contextDir string, table *gherkin.DataTable) error {
+	kogitoApp, err := getKogitoAppExamplesStub(data.Namespace, contextDir, table)
 	if err != nil {
 		return err
 	}
+	if kogitoApp.Spec.Build.Native {
```

Comment:
that is not ignored by the operator ?

Reply:
Hmm, good question. But anyway it would be good to inform a user of a framework that this configuration is illegal as it can happen that somebody will by mistake use `| config | native | true |` with Spring Boot. Wdyt?

Reply:
fair enough.
anyway once Spring will have native compilation (not today bu may come), this can be easily  removed ;)